### ISSUE TITLE
Uberjar documentation and/or support

### ISSUE THREAD

**Issue by ajoberstar** [gradle-clojure/gradle-clojure#67]
I'm guessing we can just leverage the [shadow plugin](https://github.com/johnrengelman/shadow). The question will be whether to document common use cases or apply it as part of this plugin.

**Comment by jszakmeister** [comment]
Yeah, we can definitely use it--though I'm not sure it's really the best way to go.  I fixed it a while back to honor timestamps.  It used not to and that caused some interesting issues. :-)

That's a really good question.  I guess I'd say that we document the common use-cases.  The nice part then is if someone needs to do something differently, they at least have a good starting point and aren't locked in or left thinking that the plugin's way is the only way to package it.
